Production/python/customiseHLT.py

```diff
@@ -6,6 +6,12 @@
 # file dataset=/store/mc/Run3Summer21DRPremix/TT_TuneCP5_14TeV-powheg-pythia8/GEN-SIM-DIGI-RAW/120X_mcRun3_2021_realistic_v6-v2/2540000/00c642d1-bf7e-477d-91e1-4dd9ce2c8099.root
 import FWCore.ParameterSet.Config as cms
 from PhysicsTools.NanoAOD.common_cff import Var, P4Vars
+from RecoJets.JetProducers.ak4GenJets_cfi import ak4GenJets
+from PhysicsTools.NanoAOD.jetMC_cff import Var, P4Vars
```

Comment:
Wouldn't it be a conflict with `Var/P4Vars` imported earlier? I would assume that these are two different entities

Reply:
This duplication seems to be added by mistake, thanks for pointing it out! I will remove the duplicate import.